Patches/British Weapons Set/British_weapon_set.xml

```diff
@@ -0,0 +1,773 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+<Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>British Weapon Set</li>
+    </mods>
+ <match Class='PatchOperationSequence'>
+	
+	<operations>
+  
+     <!-- Bren Gun -->
+	 
+  <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+	<defName>BWS_Bren</defName>
+	<statBases>
+		<Mass>1.3</Mass>

```

Comment:
typo?

Reply:
oh you mean mass of a weapon i assume? i didnt notice. You are right theres and error there